app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -98,6 +98,9 @@ class CatalogueController @Inject()(
   private lazy val jenkinsLinkName =
     configuration.getOptional[String]("teams-and-repositories.link-name.jenkins").getOrElse("jenkins")
 
+  private lazy val hideArchivedRepositoriesFromTeam: Boolean =
+    configuration.getOptional[Boolean]("team.hideArchivedRepositories").getOrElse(false)
```

Comment:
It's preferable to put all default values in config - i.e. it's not optional. That way you don't need to look through code to see what the behaviour would be without it, and config-explorer would also display it (although it's not wired up for platapps).

Reply:
just seen that there is already a default in `application.conf`

Reply:
@colin-lamed Addressed this.